docs/source/troubleshooting/troubleshooting-driver-plugins.adoc

```diff
@@ -0,0 +1,216 @@
+[[troubleshooting-driver-plugins]]
+= Troubleshooting Driver Plug-ins
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
```

Comment:
You need at least doc level of 2 here, otherwise the items will be impossible to find